server/routes/ticket.py

```
@@ -302,5 +300,333 @@ def post(self):
+                    "messages": formatted_messages,
+                }
+            )
+
+            return restful_response(
+                status="success",
+                message="Ticket fetched successfully",
+                data=ticket_data,
+                status_code=200,
+            )
+
+        except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)